src/tests/multihost/alltests/test_misc_proxy.py

```diff
@@ -0,0 +1,222 @@
+          2. Should not succeed
+        """
+        tools = sssdTools(multihost.client[0])
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "getent netgroup testsumgroup")
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'proxy_pam_target': 'sssdproxyldap',
+                         'id_provider': 'proxy',
+                         'proxy_lib_name': 'files',
+                         'auth_provider': 'proxy'}
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        execute_cmd(multihost, "echo 'QAeng    "
+                               "(host1.example.com, ami1, example.com)'"
+                               " > /etc/netgroup")
+        tools.clear_sssd_cache()
+        execute_cmd(multihost, "getent netgroup QAeng")
+        execute_cmd(multihost, 'echo "" > /etc/netgroup')
+        # more than 10 auth attempt times out bz647816
+        domain_params = {'proxy_lib_name': 'ldap',
+                         'debug_level': '0xFFF0'}
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        tools.clear_sssd_cache()
+        client_e = multihost.client[0].ip
+        for _ in range(12):
+            ssh1 = SSHClient(client_e, username="foo1@example1",
+                             password="Secret123")
+            ssh1.close()
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "grep 'All available child slots are full, "
+                                   "queuing request' /var/log/sssd/*")
+
+    def test_0002_bz1209483(self, multihost,
+                            backupsssdconf,
+                            create_350_posix_users):
+        """
+        :title: sssd does not work as expected when id provider
+         equal to proxy and auth provider equal to ldap bz1209483
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1209483
+        :id: 2fa4834c-b252-11ec-919d-845cf3eff344
+        :steps:
+          1. Adding the user in local
+          2. sssd runs out of available child
+          slots and starts queuing requests in proxy mode
+        :expectedresults:
+          1. Should succeed
+          2. Should not succeed
+        """
+        client_e = multihost.client[0].ip
+        tools = sssdTools(multihost.client[0])
+        # sssd does not work as expected when id provider equal to proxy
+        # and auth provider equal to ldap bz1209483
+        execute_cmd(multihost, "systemctl stop nslcd.service")
+        execute_cmd(multihost, "systemctl stop sssd")
+        execute_cmd(multihost, "useradd -u 2001 foo2")
+        execute_cmd(multihost, "echo 'pam.d/         pam_ldap.conf' > /etc/pam")
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'debug_level': '0xFFF0',
+                         'id_provider': 'proxy',
+                         'proxy_lib_name': 'files',
+                         'proxy_pam_target': 'sssdproxyldap',
```

Comment:
`proxy_pam_target` is not needed here because `auth_provider = ldap`, this is already wrong in the original test.

Reply:
done